packages/components/src/popover/index.js

```diff
@@ -188,8 +189,33 @@ const Popover = (
 		resize = ! __unstableForcePosition;
 	}
 
+	if ( anchorRef !== undefined ) {
+		// deprecated( '`anchorRef` prop in Popover component', {
+		// 	since: '6.1',
+		// 	version: '6.3',
+		// 	alternative: '`anchor` prop',
+		// } );
+	}
+
+	if ( anchorRect !== undefined ) {
+		// deprecated( '`anchorRect` prop in Popover component', {
+		// 	since: '6.1',
+		// 	version: '6.3',
+		// 	alternative: '`anchor` prop',
+		// } );
+	}
+
+	if ( getAnchorRect !== undefined ) {
+		// deprecated( '`getAnchorRect` prop in Popover component', {
+		// 	since: '6.1',
+		// 	version: '6.3',
+		// 	alternative: '`anchor` prop',
+		// } );
+	}
```

Comment:
The actual deprecation warnings have been disabled to cause less noise in the tests. I will re-enable them once all props have been migrated, as they will also help us to discover any leftover props to migrate